pyiron_atomistics/atomistics/structure/factories/materialsproject.py

```diff
@@ -139,9 +144,14 @@ def by_id(material_id: Union[str, int], api_key=None) -> Atoms:
         Args:
             material_id (str): the id assigned to a structure by the materials project
             api_key (str, optional): if your API key is not exported in the environment flag MP_API_KEY, pass it here
+            final (bool, optional): if set to False, returns the list of initial structures,
+            else returns the final structure. (Default is True)
+            conventional_unit_cell (bool, optional): if set to True, returns the standard conventional unit cell.
+            (Default is False)
 
         Returns:
-            :class:`~.Atoms`: requested structure
+            :class:`~.Atoms`: requested final structure if final is True or
+            a list of initial structures if final is False
```

Comment:
```suggestion
            :class:`~.Atoms`: requested final structure if final is True
            list of :class:~.Atoms`:  a list of initial (pre-relaxation) structures if final is False
```

is probably how I would do it,  but no hard feelings. Thanks for this timely PR!